### PR TITLE
Enable dns lookup of s-pgw ip

### PR DESCRIPTION
**Pull request by asreek**
Removed' #IF 0' skip.

### CODE REVIEW

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

File: src/s11/json_config.c [hunk 1]
```diff
@@ -88,7 +83,6 @@ parse_s11_conf()
 	  	}
 	  }
 	}
-#endif
```
Comment (thakurajayL) [inline src/s11/json_config.c:91]: Did we test it ?

Comment (asreek) [reply src/s11/json_config.c:91]: Tested using ignite.

**Comment by thakurajayL** [comment]
@sindack @yoooou  Should we not enable CI builds on this branch as well ? Dont make it mandatory but at least getting result would be great.

@pingping-lin @hariharanb77  what do you think?

**Comment by clstefi** [comment]
The commit "EDGEPOD-367-PURGE messaging during UE Init detach" involves the following changes:
1. Purge Messaging in UE Initiated Detach.
2. Static Address Support.
3. Initialization of Buffer in s1ap handlers.
4. Logging Unhandled IE in s1ap handlers.
5. Monitor config changes.

Multiple PLMN Support Changes will be pushed along with Handover Implementation.

**Pull request merged**